app/controllers/api/v1/users/omniauth_callbacks_controller.rb

```diff
@@ -27,8 +24,8 @@ def render_success(user)
           render json: { data: token_data }, status: :ok
         end
 
-        def render_error(exception)
-          render json: { error: exception.message }, status: :internal_server_error
+        def render_error(user)
```

Comment:
Self-note: Update to use method `render_error` added on the `error_handler` concern

https://github.com/nimblehq/ic-rails-huey-toby/pull/46/commits/d9f38283b49d6bf8a88c377b659ada64dbadfb69#diff-9b9c5d5d6a6266cc2ae1139314a3a72eaa8367b34eee0a66d45b0bcfc86e6172

Reply:
Related to https://github.com/nimblehq/ic-rails-huey-toby/pull/44#discussion_r1259362567